PROTO_tests/tests/index_test.py

```diff
@@ -0,0 +1,105 @@
+import pytest
+
+import arkouda as ak
+
+
+class TestIndex:
+    def test_index_creation(self):
+        idx = ak.Index(ak.arange(5))
+
+        assert isinstance(idx, ak.Index)
+        assert idx.size == 5
+        assert idx.to_list() == [i for i in range(5)]
```

Comment:
This is completely stylistic and optional

i wrote this originally so im really dissing myself, I think it's cleaner to do `list(range(5))` here instead
```suggestion
        assert idx.to_list() == list(range(5))
```